### ISSUE TITLE
Missing program aliases.

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/its#1324]
- ETTIME, alias for VTTIME.
- GCBULK, alias for GCMAIL.
- Aliases for SENDER: CHSEND, LMSEND, FSEND, SSEND, SS, PIX, SHIT, OHSHIT, RP, R, RA.
- Aliases for PR: TVDOC, TVDOCT, MUD, MUDDLE, INTRUP, PRIM, LSC, NLSC.
- Aliases for TELNET: TCPTN, TTN, NCPTN, PTN.

**Comment by larsbrinkhoff** [comment]
Maybe some more here?

https://github.com/PDP-10/its/blob/4e2ea8e4d851a0ea1f910c3001e5c2ddb9ff528c/src/sysen1/jobs.87#L66-L118

**Comment by larsbrinkhoff** [comment]
TS BOOT11 is an alias for TS 11STNK.  It's a timesharing version of .; @ BOOT11, i.e. it takes 11DDT and IOELEV and stuffs a PDP11 over DL10.

**Comment by larsbrinkhoff** [comment]
STYLOC is an alias for TTLOC.

**Comment by larsbrinkhoff** [comment]
Here's a long list of links:

http://sv.svensson.org/KLH%3BXLINKS

**Comment by eswenson1** [comment]
Any idea what CHANNA; TS SAM -> CHANNA; MAHASA MATMAN is/was?  This link exists in PI/ES but the link target is non-existent. I see no trace of "TS SAM" and MAHASA or MATMAN anywhere.

**Comment by eswenson1** [comment]
I read the one I picked up in Tallinn, but evidently all the cool ITS names came from the other. Can’t even remember the names of the books — just the name Amber. In any case, I didn’t the book compelling enough to read the other. It was ok — just not my genre.

**Comment by larsbrinkhoff** [comment]
I think most names come from Lord of Light.  Channa, Taraka, Rakash(a), and Mahasamatman if the link counts.  The Amber series is just Chaos(net) and Amber.

**Comment by eswenson1** [comment]
Yeah. I just read the first book of the Amber series. How was Lord of Light?

**Comment by larsbrinkhoff** [comment]
Amber is more fantasy-like, wereas Lord of Light is more science fiction.  The Amber series is ten books and my copy is 1200 pages so it's a more sustained effort to get through all of it.  Lord of Light has been claimed to be one of the best SF books, but if you read it with that in mind your expectations may be too high.  I think when it came out, reading about Indian techno gods was an intriguing concept.

**Comment by larsbrinkhoff** [comment]
SYS2; TS E should be a link to SYS2; TS EMACS.
